include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -259,24 +259,27 @@ struct transform_reduce
             __local_mem[__local_idx] = __res;
         }
     }
+
+    // For non-SPIR-V targets, we check if the operator is commutative before selecting the appropriate codepath.
+    // On SPIRV targets, we force the non-commutative implementation as this is currently more performant.
```

Comment:
```suggestion
    // For non-SPIR-V targets, we check if the operator is commutative before selecting the appropriate codepath.
    // On SPIRV targets, the sequential implementation with the non-commutative operator is currently more performant.
```

Reply:
I have added this suggestion (and changed SPIRV to SPIR-V to be consistent).